pkgs/by-name/wa/watchdogd/package.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, autoreconfHook, libite, libuev
+, libconfuse }:
+stdenv.mkDerivation rec {
+  pname = "watchdogd";
+  version = "3.5";
+
+  src = fetchFromGitHub {
+    owner = "troglobit";
+    repo = "watchdogd";
+    rev = "${version}";
+    hash = "sha256-DnXIOGZQK16jn20ASwrKIih/61zUuTTgeq7Gv9WLxxU=";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  buildInputs = [ libite libuev libconfuse ];
+
+  meta = with lib; {
+    description = "Advanced system & process supervisor for Linux";
+    homepage = "https://troglobit.com/watchdogd.html";
```

Let's add the changelog as well

```suggestion
    homepage = "https://troglobit.com/watchdogd.html";
    changelog = "https://github.com/troglobit/watchdogd/releases/tag/${version}";
```